src/extensions/renderer/base/arrow-shapes.js

```diff
@@ -232,15 +220,10 @@ BRp.registerArrowShapes = function(){
     },
 
     draw: function( context, size, angle, translation, edgeWidth ){
-      var scaledCrossLine = [
-        this.crossLinePoints[0] + this.scaleCrossLineXCoord( size, edgeWidth ),
-        this.crossLinePoints[1] - this.scaleCrossLineYCoord( size, edgeWidth ),
-        this.crossLinePoints[2] - this.scaleCrossLineXCoord( size, edgeWidth ),
-        this.crossLinePoints[3] - this.scaleCrossLineYCoord( size, edgeWidth )
-      ];
+      var scaledCrossLineSize = size*0.94 + edgeWidth + 2; 
       var triPts = transformPoints( this.points, size, angle, translation );
```

Comment:
I think the triPts may not be aligned to the endpoint:

![screen shot 2017-08-24 at 2 15 32 pm](https://user-images.githubusercontent.com/989043/29681759-b74a63ac-88d6-11e7-8c61-c02cb328be6c.png)

Debug page for edge style:

```
'width': 5,
 'arrow-scale': 0.25,
```